### ISSUE TITLE
Aggregations on date values return also a key 'value_as_string'

### ISSUE THREAD

**Issue by giganteous** [braedon/prometheus-es-exporter#42]
Hi Breadon. First of all: nice software: I had an itch, and I found it.

I have 2 problems, one is covered by #41, the other is that the resultset returns a value_as_string for a timestamp:

```json
{
    "size": 0,
    "query": {
            "query_string": {
                "query": "somefield:somevalue"
            }
    },
    "aggs": {
        "max_timestamp": { "max": { "field": "Timestamp" } }
    }
}
```

This returns
```json
{
....
    "hits": {
        "total": {
            "value": 10000,
            "relation": "gte"
        },
        "max_score": null,
        "hits": []
    },
    "aggregations": {
        "max_timestamp": {
            "value": 1563960734019.0,
            "value_as_string": "2019-07-24T09:32:14.019Z"
        }
    }
}
```
I can easily find the line in parse_agg() that can skip this. I can't tell of find references to why this "value_as_string" is included, ES6 and 7 both return this extra field.

Can I change my query to not return that 'value_as_string' or is this a bug in the exporter?

**Comment by giganteous** [comment]
Ah, this is the default for a date mapped field. I was too quick to name the issue. Renaming it.

**Comment by giganteous** [comment]
I tried the extended_stats aggregation, and got this back (es6):
```
{
    "_shards": {
        "failed": 0,
        "skipped": 0,
        "successful": 4,
        "total": 4
    },
    "aggregations": {
        "ext": {
            "avg": 1563960692666.1763,
            "avg_as_string": "2019-07-24 09:31:32.666",
            "count": 10604,
            "max": 1563965650440.0,
            "max_as_string": "2019-07-24 10:54:10.440",
            "min": 1563956138081.0,
            "min_as_string": "2019-07-24 08:15:38.081",
            "std_deviation": 2623278.6152724377,
            "std_deviation_as_string": "1970-01-01 00:43:43.278",
            "std_deviation_bounds": {
                "lower": 1563955446108.9458,
                "upper": 1563965939223.4067
            },
            "std_deviation_bounds_as_string": {
                "lower": "2019-07-24 08:04:06.108",
                "upper": "2019-07-24 10:58:59.223"
            },
            "sum": 1.6584239185032132e+16,
            "sum_as_string": "527503-08-15 18:57:12.132",
            "sum_of_squares": 2.593709820323737e+28,
            "sum_of_squares_as_string": "292278994-08-17 07:12:55.807",
            "variance": 6881590693345.678,
            "variance_as_string": "2188-01-26 00:58:13.345"
        }
    },
    "hits": {
        "hits": [],
        "max_score": 0.0,
        "total": 10604
    },
    "timed_out": false,
    "took": 29
}
```

I'll add a testcase for this, and a possible fix.

**Comment by braedon** [comment]
Fixed in [0.5.2](https://github.com/braedon/prometheus-es-exporter/releases/tag/0.5.2)
